index.js

```diff
@@ -42,9 +45,16 @@ const portCheckSequence = function * (ports) {
 
 module.exports = async options => {
 	let ports;
+	let exclusions = [];
 
 	if (options) {
-		ports = typeof options.port === 'number' ? [options.port] : options.port;
+		if (options.port) {
+			ports = typeof options.port === 'number' ? [options.port] : options.port;
+		}
+
+		if (options.exclusions) {
+			exclusions = typeof options.exclusions === 'number' ? [options.exclusions] : options.exclusions;
```

Comment:
There should be validation here that throws if the option is not an `Iterable<number>`.

Reply:
Done!